src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -1120,7 +1120,7 @@ export class SaveSequentializer {
 			this._nextSave.run = run;
 		}
 
-		return this._nextSave.promise;
+		return this._nextSave!.promise;
```

Comment:
@mjbvz fishy, `this._nextSave` is obviously being checked on a few lines above, suggest to file a bug against TS to be smarter here.